whole-tale/ngx-dashboard
# fix: notification stream became dewired when clicking acknowledge

## Problem
Clicking "Acknowledge All" on the notification stream would prevent future events from displaying. Only a refresh would reset the state in this case.
  
## Approach
Fix some bad patterns that may have in some way contributed to the bad state.

## How to Test
Prerequisites: at least one Tale created, stopped

1. Checkout this branch locally, rebuild the dashboard
2. Login to the WholeTale Dashboard
3. Launch a Tale
    * You should see the notification stream popup
4. Click "Acknowledge All" to dismiss the popup
5. Without refreshing, stop and launch the Tale again
    * You should see the notification stream appear again
